lighthouse-core/lib/styles-helpers.js

```diff
@@ -104,29 +104,25 @@ ${parsedContent.selector} {
 }
 
 /**
- * Returns a array of all CSS prefixes and the default CSS style names.
+ * Returns an array of all CSS prefixes and the default CSS style names.
  *
- * @param {string|Array<string>=} propNames CSS text content.
```

Comment:
I should probably allow it to use string, so I'll leave this but handle strings in the method!